src/actions/isBatchPoster.ts

```diff
@@ -0,0 +1,27 @@
+import { Address, Chain, PublicClient, ReadContractReturnType, Transport } from 'viem';
+import { sequencerInbox } from '../contracts';
+import { ActionParameters } from '../types/Actions';
+
+type Args = {
+  batchPoster: Address;
+};
+type SequencerInboxABI = typeof sequencerInbox.abi;
+export type IsBatchPosterParameters<Curried extends boolean = false> = ActionParameters<
+  Args,
+  'sequencerInbox',
+  Curried
+>;
```

Comment:
this is very similar to `GetMaxTimeVariationParameters` and `IsValidKeysetHashParameters` etc

do we really want an independent one in every file?

Reply:
Yes, you can see the reasoning here: https://linear.app/offchain-labs/issue/FS-504/explore-ways-to-simplify-contract-specific-utilities-for-v1#comment-6ca0cf13